packages/ETLoad.package/ETLObjectHeader.class/instance/hash.st

```diff
@@ -0,0 +1,7 @@
+comparing
+hash
+	^ self sizeInWords hash
```

Comment:
I only really implemented `#hash` because I implemented `#=`, and I only really implemented that, because it made testing easier. Bottom line: We don't know if `#hash` will ever actually be used.